script.js

```diff
@@ -1,4 +1,345 @@
+// Pseudo Code
+// 1. Player 1 - Upon click of submit, call diceRoll function to give 2 random numbers
+// 2. If 2 randoms numbers same, move to player 2, otherwise, let player input to choose the order
+// 3. repeat for player 2
+//. 4. see which number is bigger at the end to decide the winner
+
+// Game mode state and global variables
+var currentGameMode = "playerOne";
+var playerOneNumber;
+var playerTwoNumber;
+var selectedNumbers = [];
+var playerOneRollOne;
+var playerOneRollTwo;
+var playerTwoRollOne;
+var playerTwoRollTwo;
```

Comment:
making too many global variables can make your code extremely cluttered. try using arrays to store data that is similar! e.g var playerOneRoll = [] you can push the values from the different rounds into the array and access them using it's index